Publish via CloudCannon (#471)
CloudCannon/platform-documentation: changelogs/2025-05-13_publishing-conflict-resolution-domains-rework-and-more.mdx

* &nbsp;We have updated the appearance of the *Recommendations* task list on the *Status* page, in preparation for more Site setup suggestions.
* You can now add a Site Icon to the root folder of your Git repository using the *Upload Site Icon* button in the *Recommendations* task list.
* You can now generate an Initial Site Settings file in the `.cloudcannon` folder of your Git repository using the *Generate initial-site-settings.json* button in the *Recommendations* task list. You can also generate this file from the *Build Configuration* page, under *Site Settings*.

### Source Editor updates

This release updated the engine for our Source Editor.

* The Source Editor now has better syntax highlighting, specifically in files with YAML front matter and markdown content.
* The Source Editor now supports normal and regex search using the `CMD+F` or `CTRL+F` keyboard shortcut.
* You can now edit your files on multiple lines simultaneously. Highlight a string in the Source Editor and use the `CMD+D` or `CTRL+D` keyboard shortcut to add multiple cursors at every instance of that string in the document.

> * The new engine for our Source Editor has an expanded list of color themes. The new default theme is `basic_dark`.

Reviewer: ```suggestion
* The new engine for our Source Editor has a refined list of color themes. The new default theme is `basic_dark`.
```

Reviewer: The new list of available themes is actually smaller now

* Before this release, the *Clone* file action in the *Context Menu* would create an exact copy of the file, including any [instance values](/documentation/articles/configure-your-inputs/#instance_value) in inputs. This behavior was often undesired as multiple files would have the same UUID or the date would not reflect the time the file was created.
* The *Clone* file action is now called *Duplicate* and generates new values for instance values.

### Other features & improvements

* Renamed the *Review changes* modal to the *Save changes* modal and updated the copy on the *Save* button to better reflect their functions.
* Added support for [Google's Material Symbols](https://fonts.google.com/icons?icon.set=Material+Symbols).
* Renamed the *Conflicting updates* status to *Syncing is paused* to better reflect the app state, and removed the warning notification from the *Site navigation*.
* CloudCannon will alert you when Syncing is paused with a red warning notification over the *Save* button and a warning notification in the *Save changes* modal.

> * Improved accessibility by replacing all *Click to confirm* buttons in the app this *Confirm* modals.

Reviewer: ```suggestion
* Improved accessibility by replacing all *Click to confirm* buttons in the app with *Confirm* modals.
```
